doc/adr/0006-create-schema-lock-table-using-configuration.md

```diff
@@ -0,0 +1,42 @@
+# 6. Create schema lock table using configuration
+
+Date: 18/07/2016
```

Comment:
The .md files, including the dates, are auto-generated using [ADR tools](https://github.com/npryce/adr-tools). It doesn't appear to be possible to customise them.